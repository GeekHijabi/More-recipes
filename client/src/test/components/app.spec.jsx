import React from 'react';
import expect from 'expect';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

import App from '../../js/app';

localStorage.setItem('token', 'psdafdf');

configure({ adapter: new Adapter() });

describe('App', () => {
  it('should render routes correctly', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
