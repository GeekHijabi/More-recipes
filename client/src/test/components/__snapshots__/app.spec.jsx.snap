// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render routes correctly 1`] = `
<BrowserRouter>
  <div
    id="wrap"
  >
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        path="/signup"
      />
      <Route
        component={[Function]}
        path="/signin"
      />
      <Route
        component={[Function]}
        path="/forgot-password"
      />
      <Route
        component={[Function]}
        path="/reset-password"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/recipes"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/recipe/:id"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/profile"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/favorites"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/admin"
      />
      <Route
        component={[Function]}
      />
    </Switch>
  </div>
</BrowserRouter>
`;
